src/js/feedback/components/FeedbackForm.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import AlertBox from '../../common/components/AlertBox';
+import ErrorableTextarea from '../../common/components/form-elements/ErrorableTextarea';
+import ErrorableTextInput from '../../common/components/form-elements/ErrorableTextInput';
+
+function FeedbackForm(props) {
+  return (
+    <form
+      className="feedback-form"
+      onSubmit={(event) => {
+        event.preventDefault();
+        props.sendFeedback();
+      }}>
+      {props.errorMessage ? (
```

Comment:
This is ultimately up to the designers, but our longer forms tend to put these global errors below the submit buttons (less chance of missing them because of scroll position).